test/regression.py

```diff
@@ -117,7 +117,7 @@ def testADF_ADF2013_01_stopiter_dvb_sp_b_adfout(logfile):
 
 def testADF_ADF2013_01_stopiter_dvb_sp_c_adfout(logfile):
     """This logfile has not SCF test lines so we have no way to check what happens."""
-    # This is what we would have checked:
+    # This is what we ADFwould have checked:

```

Comment:
Something weird happened here...

Reply:
Apologies, this typo is now addressed. Thank you for pointing out the git config oddity. I hopefully have addressed that as well, but let me know!

Reply:
Sure, if you're up for it.